### PR TITLE
Reorganize docs structure and reduce duplicate

### PR DESCRIPTION
**Pull request by jonchenn**
- Reorganize docs structure 
- Reduce duplicate docs in the root vs. cookecutter folder.
- Adding CONTRIBUTING with development steps.
- Leave other docs for improvement later.

### CODE REVIEW

File: CONTRIBUTING [hunk 1]
```diff
@@ -1,6 +1,82 @@
...
+```
+git commit -a -m 'your message about this change'.
+```
+
+### Create a Pull Request for review
+
+Create a [Pull Request](https://github.com/GoogleCloudPlatform/solutions-template/compare) to merge to `main` branch, and tag the project owners/admins to review.
+
+- The project owners and admins will run end-to-end tests (TBD) to ensure the change doesn't break existing functions.
+
```
Comment (lavjain) [inline CONTRIBUTING]: This file should be renamed to `CONTRIBUTING.md`

Comment (lavjain) [reply CONTRIBUTING]: `skaffold` version is not required when downloading. `LATEST` should work:
```
  curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/LATEST/skaffold-linux-amd64 && \
```

File: docs/DEVELOPMENT.md [hunk 2]
```diff
@@ -35,14 +35,24 @@
...
 
-This doc explains the development workflow so you can get started contributing code to {{cookiecutter.project_name}}
+This doc explains the development and code submission process.
 
 
 ##  1. <a name='ProjectRequirements'></a>Project Requirements
 
+### Tool requirements:
+
 Install the following based on [the versions for this project](./README.md#ProjectRequirements)
```
Comment (lavjain) [inline docs/DEVELOPMENT.md]: The above hyperlink does not work

File: CONTRIBUTING (hunk 1 above)
Comment (jonchenn) [reply CONTRIBUTING]: file renamed.

File: docs/INSTALLATION.md [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+  ```
+
+###  1.4. <a name='AppleM1ChipSupportforTerraformOptional'></a>Apple M1 Chip Support for Terraform (Optional)
+
+If you are running commands on an Apple M1 chip Macbook, make sure run the following to add M1 support for Terraform:
+```
+brew install kreuzwerker/taps/m1-terraform-provider-helper
+m1-terraform-provider-helper activate
+m1-terraform-provider-helper install hashicorp/template -v v2.2.0
+```
```
Comment (lavjain) [inline docs/INSTALLATION.md:86]: The info above is already mentioned in TROUBLESHOOTING.md

File: CONTRIBUTING (hunk 1 above)
Comment (jonchenn) [reply CONTRIBUTING]: We had an issue of Skaffold v2 where it dramatically changed the manifest and broke the deployment. I'd prefer keep the version to 2.0.4 at least, so it's stable and working until we test with newer version.

File: docs/INSTALLATION.md [hunk 4]
```diff
@@ -0,0 +1,398 @@
+# Solutions Template - Installation
+
+Table of Content
+
+<!-- vscode-markdown-toc -->
+* 1. [Prerequisites](#Prerequisites)
+	* 1.1. [Understanding Google Cloud](#UnderstandingGoogleCloud)
+	* 1.2. [Tool requirements:](#Toolrequirements:)
```
Comment (lavjain) [inline docs/INSTALLATION.md:8]: A few bullets not hyperlinked correctly in TOC

File: docs/DEVELOPMENT.md (hunk 2 above)
Comment (jonchenn) [reply docs/DEVELOPMENT.md]: Fixed.

File: docs/INSTALLATION.md (hunk 3 above)
Comment (jonchenn) [reply docs/INSTALLATION.md:86]: It's intentional, when users walk through the installation, these are the minimum steps they'll need to run when using Mac.

File: docs/INSTALLATION.md (hunk 4 above)
Comment (jonchenn) [reply docs/INSTALLATION.md:8]: Updated TOC

**Review by lavjain — APPROVED** [review]
Thanks a lot for re-organizing the docs!

**Pull request merged**